src/main/java/duke/parts/Parser.java

```diff
@@ -0,0 +1,42 @@
+package duke.parts;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.PrintCommand;
+import duke.error.UnknownAction;
+
+
+/**
+ * Used to parse inputs from the user
```

Comment:
I think this comment could perhaps be rephrases to be aligned with the Java Docs principle 😬
Perhaps you would want to rephrase it to `Parses input from the user`?